python/ray/serve/_private/router.py

```diff
@@ -995,12 +1002,6 @@ async def assign_request(
         result = await self._replica_scheduler.assign_replica(query)
 
         self.num_queued_queries -= 1
```

Comment:
This line raises an exception. `self.num_queued_queries` is a `DefaultDict`, so you can't `-=` it.

Reply:
oops, fix.